saladItem.js

```diff
@@ -1,3 +1,68 @@
 import React, { useReducer, useContext } from 'react';
 import PropTypes from 'prop-types';
 import { createUseStyles } from 'react-jss';
+
+import UserContext from '../User/User';
+import { SaladContext } from '../SaladMaker/SaladMaker';
+
+// Items for a list of Salad options for the Salad Maker Interface
+const useStyles = createUseStyles({
+        add: {
+          background: 'none',
+          border: 'none',
+          cursor: 'pointer',
+        },
+            favorite: {
+            fontSize: 20,
+            position: 'absolute',
+            top: 10,
+            right: 10,
+            },
+        image: {
+          fontSize: 80
+        },
+    wrapper: {
+        border: 'lightgrey solid 1px',
+        margin: 20,
+        padding: 25,
+        position: 'relative',
+        textAlign: 'center',
+        textTransform: 'capitalize',
+        width: 200,
+    }
+});
+
+var reducer = 0;
+const reducer = key => key + 1;
```

Comment:
This is a very simple incrementor function and doesn't account for a decrement or a reset of the value. I think it would be a good idea to refactor this function to include these functions.

I pulled this from the React Hooks API Reference docs, and think it would work for this application:
`function reducer(state, action) {
  switch (action.type) {
    case 'increment':
      return {count: state.count + 1};
    case 'decrement':
      return {count: state.count - 1};
    case 'reset':
      return init(action.payload);
    default:
      throw new Error();
  `}`